validators/terravaloper1fhx7y75643tze8dxf4m9gwhkxn955q8r7vxjel/profile.json

```diff
@@ -0,0 +1,9 @@
+{
+    "contact": { // contact info for delegators / outreach
```

Comment:
JSON can't have comments. please remove it.

Reply:
removed comments